examples/twittermap/web/app/controllers/TwitterMapApplication.scala

```diff
@@ -176,6 +189,41 @@ object TwitterMapApplication {
     }
   }
 
+  /**
+    * Copy and revised based on [[play.api.libs.streams.ActorFlow.actorRef()]]
+    * The difference is that instead of using implicit '''ActorRefFactory''' to instantiate actors,
+    * we use the '''ActorSystem''' parameter that passed to the method.
+    * The reason is that we don't want the implicit '''ActorRefFactory''' here because it has been used in CloudBerry and
+    * TwitterMap application should have an independent ActorSystem.
+    */
+  def actorRef[In, Out](system: ActorSystem, props: ActorRef => Props, bufferSize: Int = 16, overflowStrategy: OverflowStrategy = OverflowStrategy.dropNew)
```

Comment:
should it be reused from `zion`?

Reply:
I had seen the code snippet of `actorRef` in `zion` and I didn't think they are handling the same problem. In `TwitterMap`, the pain point is that we wanna use another `actorSystem` different from implicit one and since the default `actorRef` relying on that, we have to change it. In `zion`, I am not completely understood why we implement that but seems it still uses implicit `actorRefFactory`. That is why we cannot reuse it.

Reply:
I got it now. 
But can you check if we can directly use the system `play.api.libs.streams.ActorFlow.actorRef()` without have to write our own?

Reply:
If we want to use a different actor system other than implicit, then we cannot use the default `ActorFlow.actorRef()` since it relies on implicit `actorSystem`